### ISSUE TITLE
CMD+clicking the View Post link that appears post-publish doesn't open in a new tab

### ISSUE THREAD

**Issue by p-jackson** [Automattic/wp-calypso#55785]
By default cmd+clicking a link will open in a new tab. But doing this in the block editor on the View Post links doesn't work. So the appears to be something interferring with the default link click handling (right-clicking and opening in a new tab still works)

![image](https://user-images.githubusercontent.com/5665959/129976359-12f4613c-663b-44f1-9d24-515e7d58f124.png)

<img width="246" alt="Screen Shot 2021-08-27 at 3 06 55 pm" src="https://user-images.githubusercontent.com/22446385/131074791-b447648b-5b6c-4daa-b50c-6006608cc931.png">

Both the post-publish sidebar and the toast notification are affected.

This should probably be fixed in Gutenberg

**Comment by jordesign** [comment]
Good catch! Agreed we should respect the default behaviour for cmd+click here - regardless of the decision made in #55599

**Comment by arthur791004** [comment]
Quick note for this one: The reason for cmd + click not work is because `wpcom-block-editor` binds the click listener [here]( https://github.com/Automattic/wp-calypso/blob/e73f5e54a944f92f78d644431f3e8fa2ba45ec59/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js#L675).

`.post-publish-panel__postpublish .components-panel__body.is-opened a` and `.components-panel__body.is-opened .post-publish-panel__postpublish-buttons a.components-button` use `preventDefault` so that the cmd + click not work on any link in post publish panel. Also, the last two make the cmd + click not work on the toast notification.

**Comment by p-jackson** [comment]
Good digging! So it's a wpcom issue then. I wonder if we can do something like check whether modifier keys have been held down before running our logic.
@lsl added some global event listeners recently that modified the element attributes on the fly. I wonder if instead of calling `calypsoPort.postMessage()` on click, we could instead add a `target=_parent` or something to the `<a>` element within the click handler, and that way the default handling would open the link outside of the iframe.

**Comment by arthur791004** [comment]
Oh, I just try to ignore if the click has a modifier.

```js
// Ignore if the click has modifier
if ( e.shiftKey || e.ctrlKey || e.metaKey ) {
	return;
}
```

But I think adding a `target=_parent` is also a good choice!

**Comment by lsl** [comment]
> @lsl added some global event listeners recently that modified the element attributes on the fly

#54199 for ref, this won't be merged.

**Comment by p-jackson** [comment]
This fix was merged to calypso in #56014 and the update to `wpcom-block-editor` was shipped in D66394-code
